docs/Running/Publish.md

```diff
@@ -0,0 +1,83 @@
+# Publish & Deploy Apps
+
+Enarx is more than just a tool for running WebAssembly. It also allows you to publish a WebAssembly package to a remote package host, as well as securely run a published package in a local Enarx keep. The following sections will demonstrate this process, along with other useful commands for interacting with a package host.
+
+## Registering a user and authenticating with the package host
+
+Before we begin, you will need a GitHub account in order to authenticate to the package host.
+
+First, register a new user with the package host using the `enarx user register` command, as shown here:
+
+```
+enarx user register your_username
+```
+
+Upon entering this command, Enarx will print two things to your terminal: a web page URL, and a unique one-time code. Open the URL, then copy and paste the one-time code into the form field on the web page. Once you have entered the code, it will ask to connect to your GitHub account. Follow the instructions shown on the page, and when you have finished connecting your GitHub account you may close the page and return to your terminal. Note that it may take a few moments before your terminal finishes authenticating with GitHub. Once it does, it will notify you that your credentials have been saved locally. The command will then exit, indicating that your user account has been successfully registered.
```

Comment:
We should not here that the link should can be opened on a different device than the one on which the login command is run, e.g. when logging in on remote, headless hosts

Reply:
Done.